committee/committee.go

```diff
@@ -201,6 +201,9 @@ func (ec *committee) Start(ctx context.Context) (err error) {
 	if err = ec.archive.Start(ctx); err != nil {
 		return err
 	}
+	if ec.latestHeightInArchive() >= ec.ceilingHeight {
+		return nil
+	}
```

Comment:
just put before 201 and `ec.terminated = true`

Reply:
at least add `ec.terminated = true`

Reply:
latestHeightInArchive() calls ec.archive.TipHeight(), so need archive.Start() first?

ok, will add